multi_press/automation/jsr223/python/community/multi_press/multi_press.py

```diff
@@ -0,0 +1,178 @@
+"""
+Copyright September 23, 2020 Sascha Volkenandt
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import traceback
+import core
+from core.osgi import register_service, unregister_service
+from core.log import logging, LOG_PREFIX
+from java.util.concurrent import TimeUnit
+from org.eclipse.smarthome.core.library.types import StringType
```

Comment:
How does this run? Is `scope` imported? I've been told it's better to import `scope` and then reference scope.StringType. For one thing, that means that this code will be compatible with OH 2.5 and OH 3. If you explicitely import from org.eclipse it won't work in OH 3 as all those packages are gone.